backend/Friends/models.py

```diff
@@ -4,14 +4,9 @@
 from .managers import FriendsManager
 
 # Create your models here.
-class Friends(models.Model):
-	# User who initiated the friend request
-	user_id = models.ForeignKey(User, related_name='user_id', null=False, blank=False, on_delete=models.CASCADE)
-
-	# User who received the friend request
-	friend_id = models.ForeignKey(User, related_name='friend_id', null=False, blank=False, on_delete=models.CASCADE)
-
-	# Date and time when the friend request is sent
+class Friend(models.Model):
+	user_id = models.IntegerField()
+	friend_id = models.IntegerField()
```

Comment:
ForeingKey made it so that we actually need to get the whole model of User and Friend through the Rest API which to me didnt seem liek a good idea, so changed these to integer fields. We need to add the check for the relation between actual users from the database later but didnt do it in this PR.

Reply:
Have you made it an issue yet, just so we don't forget it?

Reply:
I was supposed to yesterday but did not, here we go: https://github.com/dorianjagusch/ft_transcendence/issues/120